autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionInfo.tsx

```diff
@@ -12,6 +12,11 @@ import meetingsForSection from '../../../../../../utils/meetingsForSection';
 import MeetingTypeDisplay from './MeetingType/MeetingTypeDisplay';
 import InstructionalMethodIcon from './InstructionalMethodIcon/InstructionalMethodIcon';
 import * as styles from './SectionSelect.css';
+import Section from '../../../../../../types/Section';
+
+export function generateSectionInfoID(section: Section): string {
+  return `${section.id}-0`;
+}
```

Comment:
```suggestion
export function generateSectionInfoID(section: Section): string {
  return `${section.id}-0`;
}
```
Is there a reason you add `-0` to the end rather than doing something like `section-info-${section.id}`? I'd prefer we keep ids helpful, the current names seem arbitrary

Reply:
I had originally put it there as a dummy, as I intended to have that there and count how many times each section ID occurred so we could have two course cards of the same type. I realized that was difficult to do and not necessary, and this is just left over.